### ISSUE TITLE
[Actions] With action connected to both gamepad and keyboard, action fires only once

### ISSUE THREAD

**Issue by Rene-Damm** [Unity-Technologies/InputSystem#425]
https://forum.unity.com/threads/input-system-update.508660/page-9#post-4229821

**Comment by Rene-Damm** [comment]
Possibly a duplicate of https://github.com/Unity-Technologies/InputSystem/issues/346.

**Comment by JiaUnity** [comment]
I have a xbox controller plugged in all the time. I've never seen this bug before.
I'll see if the user can provide more specific info.

**Comment by JiaUnity** [comment]
I've never seen this issue. No repro on 2019.1.0b9 with Develop (7170868)
